beetsplug/replaygain.py

```diff
@@ -147,12 +153,18 @@ def __init__(self, config, log):
 
         cmd = 'bs1770gain'
         try:
-            call([cmd, "--help"])
+            version_out = call([cmd, '--version'])
             self.command = cmd
+            self.version = re.search(
+                '([0-9]+.[0-9]+.[0-9]+), ',
+                version_out.stdout.decode('utf-8')
+            ).group(1)
         except OSError:
             raise FatalReplayGainError(
                 u'Is bs1770gain installed?'
             )
+        except AttributeError:
```

Comment:
Thanks for the reorganization, but I'm still not 100% sure when this exception can be raised. Isn't it the case that the `command_output` object will always have a `stdout` member if the call was successful?

Reply:
There is an edge case in `test_replaygain` where the backend is tested while bypassing calls to `bs1770gain`.
It was patching `None` to stdout there, I made it patch `bs1770gain 0.0.0, ` instead.